.gitignore

```diff
@@ -82,4 +82,6 @@ data.ms/*
 auth.json
 
 /packages/ux-shared/
-/images
\ No newline at end of file
+/images
+
+!client/src/components/Nav/SettingsTabs/Data/
```

Comment:
I have reorganized all the settings tabs, but it seems that "data/" is being ignored. Therefore, I added `!client/src/components/Nav/SettingsTabs/Data/` to include `Data.tsx` in commits